src/ApiPort/CommandLine/AnalyzeOptions.cs

```diff
@@ -88,6 +88,9 @@ public AnalyzeCommandLineOption(Options options)
                 TargetMapFile = options.TargetMap;
                 BreakingChangeSuppressions = options.SuppressBreakingChange;
 
+                //Set OverwriteOutputFile to true if the output file name is explicitly specified 
+                OverwriteOutputFile = string.IsNullOrWhiteSpace(options.Out) ? false : true;
```

Comment:
Replace with: `OverwriteOutputFile = !string.IsNullOrWhiteSpace(options.Out);`
It is possible that even though we set this to not overwrite that the OutputFileName will still be " ".  Can we move this error checking into a single location and consolidate [AnalyzeOptions:Line85](https://github.com/alinapopa/dotnet-apiport/blob/d1fc4354c66929717a64b6497a06ab4a60a82280/src/ApiPort/CommandLine/AnalyzeOptions.cs#L85) into this:

```csharp
// Should we bother checking for invalid path characters (using System.IO.Path.GetInvalidPathChars()) 
// or wait for it to fail later when it tries to write to the file system?
if (!string.IsNullOfWhiteSpace(options.Out))
{
   OverwriteOutputFile = true;
   OutputFileName = options.Out;
}
```